app/src/main/java/com/sopt/clody/ui/theme/Type.kt

```diff
@@ -1,34 +1,129 @@
-    )
-    /* Other default text styles to override
-    titleLarge = TextStyle(
-        fontFamily = FontFamily.Default,
-        fontWeight = FontWeight.Normal,
-        fontSize = 22.sp,
-        lineHeight = 28.sp,
-        letterSpacing = 0.sp
     ),
-    labelSmall = TextStyle(
-        fontFamily = FontFamily.Default,
+    body1SemiBold = pretendardTextStyle.copy(
+        fontSize = 15.sp,
+        lineHeight = 22.5.sp,
+    ),
+    body1Medium = pretendardTextStyle.copy(
+        fontWeight = FontWeight.Medium,
+        fontSize = 15.sp,
+        lineHeight = 22.5.sp,
+    ),
+    body2SemiBold = pretendardTextStyle.copy(
+        fontSize = 14.sp,
+        lineHeight = 21.sp,
+    ),
+    body2Medium = pretendardTextStyle.copy(
         fontWeight = FontWeight.Medium,
-        fontSize = 11.sp,
-        lineHeight = 16.sp,
-        letterSpacing = 0.5.sp
+        fontSize = 14.sp,
+        lineHeight = 21.sp,
+    ),
+    body3SemiBold = pretendardTextStyle.copy(
+        fontSize = 13.sp,
+        lineHeight = 19.5.sp,
+    ),
+    body3Medium = pretendardTextStyle.copy(
+        fontWeight = FontWeight.Medium,
+        fontSize = 13.sp,
+        lineHeight = 19.5.sp,
+    ),
+    detail1SemiBold = pretendardTextStyle.copy(
+        fontSize = 12.sp,
+        lineHeight = 18.sp,
+    ),
+    detail1Medium = pretendardTextStyle.copy(
+        fontWeight = FontWeight.Medium,
+        fontSize = 12.sp,
+        lineHeight = 18.sp,
+    ),
+    detail2SemiBold = pretendardTextStyle.copy(
+        fontSize = 10.sp,
+        lineHeight = 15.sp,
+    ),
+    detail2Medium = pretendardTextStyle.copy(
+        fontWeight = FontWeight.Medium,
+        fontSize = 10.sp,
+        lineHeight = 15.sp,
+    )
+)
+
+@Immutable
+data class ClodyTypography(
+    val head1: TextStyle,
+    val head2: TextStyle,
+    val head3: TextStyle,
+    val head4: TextStyle,
+    val head5: TextStyle,
+    val body1SemiBold: TextStyle,
+    val body1Medium: TextStyle,
+    val body2SemiBold: TextStyle,
+    val body2Medium: TextStyle,
+    val body3SemiBold: TextStyle,
+    val body3Medium: TextStyle,
+    val detail1SemiBold: TextStyle,
+    val detail1Medium: TextStyle,
+    val detail2SemiBold: TextStyle,
+    val detail2Medium: TextStyle
+)
+
+val LocalTypography = staticCompositionLocalOf {
+    ClodyTypography(
+        head1 = pretendardTextStyle,
+        head2 = pretendardTextStyle,
+        head3 = pretendardTextStyle,
+        head4 = pretendardTextStyle,
+        head5 = pretendardTextStyle,
+        body1SemiBold = pretendardTextStyle,
+        body1Medium = pretendardTextStyle,
+        body2SemiBold = pretendardTextStyle,
+        body2Medium = pretendardTextStyle,
+        body3SemiBold = pretendardTextStyle,
+        body3Medium = pretendardTextStyle,
+        detail1SemiBold = pretendardTextStyle,
+        detail1Medium = pretendardTextStyle,
+        detail2SemiBold = pretendardTextStyle,
+        detail2Medium = pretendardTextStyle
     )
```

Comment:
텍스트가 다 동일하게 보이는 문제점을 찾았습니다.
`pretendardTextStyle`로 초기화 되서 똑같이 보이는 문제가 있었습니다.
```suggestion
val LocalTypography = staticCompositionLocalOf {
    defaultClodyTypography
}
```